### PR TITLE
RDAPI-24811 - Updated Cassandra Upgrade guide for version 3.11.11

### PR DESCRIPTION
**Pull request by credmond-axway**
Added a new migration guide for upgrading from Cassandra 2.2.12 to 3.11.11. Also updated the System Properties page with details of the new Cassandra protocol configuration.

## Deploy preview link

https://deploy-preview-1990--axway-open-docs.netlify.app/docs/apim_installation/apigw_upgrade/upgrade_cassandra/
https://deploy-preview-1990--axway-open-docs.netlify.app/docs/apim_reference/system_props/

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by sgoka-axway — APPROVED** [review]
Sounds good!

**Comment by matthewboss-axway** [comment]
Thanks, @credmond-axway, this looks very clear (but admittedly I've only read through it quickly).

This might be a bit tangential to purely "documentation" questions, but I note that these are the zero-downtime instructions. Will there be "with downtime" instructions, and would this allow customers to skip 2.2.19? A lot of them will balk at having to upgrade "twice". They struggle with upgrades quite enough as things are.

**Comment by credmond-axway** [comment]
> Thanks, @credmond-axway, this looks very clear (but admittedly I've only read through it quickly).
> 
> This might be a bit tangential to purely "documentation" questions, but I note that these are the zero-downtime instructions. Will there be "with downtime" instructions, and would this allow customers to skip 2.2.19? A lot of them will balk at having to upgrade "twice". They struggle with upgrades quite enough as things are.

Thanks @matthewboss-axway . I agree. I've added details of how to upgrade directly to 3.11.11 (i.e. bypass 2.2.19) which will include downtime.

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra.md [hunk 1]
```diff
@@ -1,338 +1,342 @@
...
-* `/opt/db/cassandra-228/cassandra/conf/.keystore`
+```
+./startinstance -g "my_group" -n "my_server" -k
+```
 
-### Update Cassandra 2.2.12 configuration files
+### Step 6 - Shutdown Cassandra 2.2.8/2.2.12
 
-Update your Cassandra 2.2.12 configuration files with the relevant settings from your Cassandra 2.2.8 installation. The following files must be updated:
+1. Run Cassandra's `nodetool drain` command in `CASSANDRA_HOME/bin` to clear data from the node and isolate it from the cluster.
```
Comment (kmercierwhite) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra.md:82]: Reword this to

1. Run Cassandra's `nodetool drain` command in `CASSANDRA_HOME/bin` to flush any data in memory and write it to disk.

Comment (kmercierwhite) [reply content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra.md:82]: And also do this to the same lines below

Comment (credmond-axway) [reply content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra.md]: Thanks @kmercierwhite. I've updated that now.

**Review by kmercierwhite — APPROVED** [review]
